Assignment/Program.cs

```diff
@@ -1,7 +1,17 @@
-﻿namespace Assignment {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello world!");
+﻿for (; ; ) {
+   Console.Write ("Enter number to be checked :");
+   if (int.TryParse (Console.ReadLine (), out int num)) {
+      int count = 0;
+      for (int i = 2; i < num; i++) {
+         if (num % i != 0)
+            count++;
+         else {
+            Console.Write ($"{num} is not a prime number");
+            break;
+         }
+         if (count == num - 2)

```

Comment:
If it is not divisible by a number then continue and if it is divisible you break saying it is not a prime number.
I don't understand why you need this count == num - 2.